app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_ls = command.split()
+    if len(command_ls) == 3:
+        command, old_file, new_file = command.split()
```

Comment:
You can just unpack command_ls afterwards (using `*`), without performing split again